app/app.go

```diff
@@ -514,6 +514,8 @@ func New(
 	app.ClaimKeeper = *claimmodulekeeper.NewKeeper(
```

Comment:
I think if `claimmodulekeeper.NewKeeper` don't return a pointer , then here we don't need to de-pointer it

Reply:
This is the default from scaffolding, in general I am bit confused why we aren't more generally passing around pointers in the codebase, so am curious why your preference would be to pass this by value?

Reply:
Yup, I understand it is default from scaffolding , it all depends on how we use it , here we de-pointer it, so instead we could just return by value

Reply:
okay, will revise.